src/packages/core/index.ts

```diff
@@ -46,6 +61,18 @@ const Ganache = {
   ): ConnectorsByName[T]["provider"] => {
     const loader = ConnectorLoader.initialize<T>(options);
     return loader.connector.provider;
+  },
+  /**
+   * 
+   * @experimental
+   */
+  __experimental_info(): _ExperimentalInfo {
+    return {
+      version,
+      fork: {
+        knownChainIds: Array.from(KNOWN_CHAINIDS)
```

Comment:
The method seems good enough for me!  Earlier you made it sound like there might be a more expansive thing with more info about the various chains, which might be nice for later, but if that's a bunch more work, well, `knownChainIds` does the job! :)

Reply:
Oh yeah, we could potentially stuff a whole bunch of interesting stuff in here in the future.